hw6_task_3.py

```diff
@@ -0,0 +1,21 @@
+import random
+
+if __name__ == "__main__":
+
+    def is_prime():
+        var = random.randint(2, 1000)
+        if var == 2 or var == 3:
+            return var, True
+        if var % 2 == 0 or var % 3 == 0 or var % 4 == 0 or var % 5 == 0 or var % 6 == 0 or var % 7 == 0 or var % 8 == 0 or var % 9 == 0:
+            return var, False
+        for i in range(4, 1001):
+            if var % i == 0:
+                return var, False
+            else:
+                return var, True
+
+
+    res, var = is_prime()
+    print(res)
+    print(var)
```

Comment:
![image](https://user-images.githubusercontent.com/25909810/217602259-35c67a7f-aeee-4e2b-bd6d-3ae9603b23c0.png)
7 is a prime number, your code return false